### PR TITLE
Add some context to the ValidationError message.

### PR DESCRIPTION
**Pull request by Natim**
Fixes mozilla-services/amo2kinto#78

### CODE REVIEW

File: aws_lambda.py [hunk 1]
```diff
@@ -75,16 +82,18 @@ def validate_signature(event, context):
...
         except Exception as e:
             exception = e
-            print('Signature KO.')
-            print(' - Computed hash: `{}`'.format(computed_hash))
-            print(' - Collection timestamp: `{}`'.format(timestamp))
-            print(' - Serialized content: `{}`'.format(serialized))
+            message = 'Signature KO.\n'
+            message += ' - Computed hash: `{}`\n'.format(computed_hash)
+            message += ' - Collection timestamp: `{}`\n'.format(timestamp)
+            message += ' - Serialized content: `{}`\n\n'.format(serialized)
```
Comment (leplatrem) [inline aws_lambda.py]: nit: multiline within paranthesis?

File: aws_lambda.py [hunk 2]
```diff
@@ -75,16 +82,18 @@ def validate_signature(event, context):
...
+            print(message)
+            messages += message
```
Comment (leplatrem) [inline aws_lambda.py]: nit: maybe using a list and a join `\n` instead?

File: requirements.txt [hunk 3]
```diff
@@ -39,16 +39,16 @@ python-dateutil==2.6.1
...
 transaction==2.1.2
 translationstring==1.3
 ujson==1.35
-Unidecode==0.4.21
+Unidecode==1.0.22
 urllib3==1.22
 venusian==1.1.0
 waitress==1.1.0
-WebOb==1.7.3
+WebOb==1.8.0rc1
```
Comment (glasserc) [inline requirements.txt]: I thought we weren't including this?

Comment (Natim) [reply requirements.txt]: Yes my bad

**Pull request merged**